frontend/src/components/Sidebar/prepareRoutes.ts

```diff
@@ -122,11 +122,13 @@ function prepareRoutes(t: (arg: string) => string) {
   ];
 
   const items = store.getState().ui.sidebar.entries;
+  const filters = store.getState().ui.sidebar.filters;
   // @todo: Find a better way to avoid modifying the objects in LIST_ITEMS.
   const routes: SidebarItemProps[] = JSON.parse(JSON.stringify(LIST_ITEMS));
 
   for (const i of Object.values(items)) {
     const item = _.cloneDeep(i);
+
```

Comment:
small nitpick, this empty line was not there earlier

Reply:
Thanks. This is fixed.